bump_rpm.sh

```diff
@@ -62,23 +75,7 @@ if [[ $CURRENT_VERSION != "$NEW_VERSION" ]] ; then
 		cd "$1"
 		GEM_NAME=$(awk '/^%global\s+gem_name/ { print $3 }' $SPEC_FILE)
 
-		spectool --list-files $SPEC_FILE | cut -d' ' -f2 | grep http | xargs --no-run-if-empty -n 1 basename | xargs --no-run-if-empty git rm
```

Comment:
won't this leave the old gem in the annex?

Reply:
I've moved those to bump_spec, a custom function that wraps rpmdev-bumpspec

Reply:
Ah, mhh, must have been blind (or not caffeinated).